### PR TITLE
added documentation for new rule

### PR DESCRIPTION
**Pull request by yyang08**

### CODE REVIEW

**Comment by yyang08** [comment]
failling tests due to changing error message format, still in progress

File: docs/source/rules/examples/REQ-E004/new 2.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+swagger: '2.0'
```
Comment (macisamuele) [inline docs/source/rules/examples/REQ-E004/new 2.yaml]: Is this file needed? I don't see it linked anywhere

File: docs/source/rules/examples/REQ-E004/old 2.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+swagger: '2.0'
```
Comment (macisamuele) [inline docs/source/rules/examples/REQ-E004/old 2.yaml]: is this file needed? Is it possible that you've added in the repo files that should not be committed?

File: docs/source/rules/examples/REQ-E004/tester 2.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```
Comment (macisamuele) [inline docs/source/rules/examples/REQ-E004/tester 2.py]: ditto

File: swagger_spec_compatibility/rules/added_enum_value_in_response.py [hunk 4]
```diff
@@ -35,4 +34,10 @@ def validate(cls, left_spec, right_spec):
                 continue
             if not is_path_in_top_level_paths(response_paths, enum_values_diff.path):
                 continue
-            yield cls.validation_message(format_path(enum_values_diff.path))
+            message = '\n \t\t{} {}: new enum value {} in property `{}`\n\t\t'.format(
```
Comment (macisamuele) [inline swagger_spec_compatibility/rules/added_enum_value_in_response.py]: Please keeps PR as small and scoped as possible, it simplifies review and increases confidence around the change.
From what I read in the PR title we should be focusing on the creation of a new rule, and not on updating the wording of an already existing rule.

PS: removing this change would probably help you out with having green tests.

**Comment by yyang08** [comment]
@macisamuele  Thanks for the feedback. Now I'm getting back to this - will close this one and split the change into 2 separate PRs as suggested.
